### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by username-forGithub**
- Understand the concept of state management.
- Use store, actions and reducers in React.
- Connect React and Redux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# Bookstore: add reducers and actions
```
Comment (OLIPLICHE) [inline README.md:2]: ## Optional
- Please provide the concerns files to be reviewed, here in your PR. This allows reviewers to make inlines comments to your codes

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+# Bookstore: Use Redux in React components.
```
Comment (OLIPLICHE) [inline README.md:2]: ## A button for checking the status
- "Check status" button click should return "Under Construction"
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @username-forGithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Adding and Removing are working👏👏👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by username-forGithub** [comment]
Sorry I couldn't figure out how to provide files here.
But now clicked button is returning "under construction".

**Review by rotimiazeez — APPROVED** [review]
# Hi @username-forGithub 👍🏾, 
## Good job implementing the required change(s)
### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**